src/main/java/wolox/training/models/Book.java

```diff
@@ -0,0 +1,115 @@
+package wolox.training.models;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+
+@Entity
+public class Book {
+    @Id
+    private String genre;
+    @Column(nullable = false)
+    private String autor;
+    @Column(nullable = false)
+    private String image;
+    @Column(nullable = false)
+    private String title;
+    @Column(nullable = false)
+    private String subtitle;
+    @Column(nullable = false)
+    private String publisher;
+    @Column(nullable = false)
+    private String year;
+    @Column(nullable = false)
+    private Integer pages;
+    @Column(nullable = false)
```

Comment:
In these cases where the fields need to be unique ,I would not just add @Column (nullable = false)
I would also add the following:
@Column (nullable = false, unique = true)

```suggestion
    @NotNull
    @Column (nulleable = false, unique = true)
    private String isbn;
```

Reply:
I added unique in the isbn field. In this case only isbn.